src/main/java/com/bitheads/braincloud/client/BrainCloudWrapper.java

```diff
@@ -738,9 +736,10 @@ public void resetEmailPassword(String email,
      * Note that the password sent from the client to the server is protected
      * via SSL.
      *
-     * @param email       The e-mail address of the user
-     * @param serviceParams
-     * @param callback    The callback handler
+     * @param email         The e-mail address of the user
+     * @param serviceParams Parameters to send to the email service. see documentation for full
+     *                      list. http://getbraincloud.com/apidocs/apiref/#capi-mail
```

Comment:
I guess we'll have to update these whenever our new version of the API docs is published. Just something to remember/keep-in-mind.